backend/uchicago/watcher_job.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
```

Comment:
This file isn't needed, I'll set up the cronjob once this is submitted. :)

Reply:
Sounds good. :)